OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -145,6 +147,12 @@ public MiniParseEventSource(TinyIoCContainer container) : base(container)
                 });
             });
 
+            RegisterEventHandler("getPartyList", (msg) =>
+            {
+                var partyList = partyListMemory.GetPartyList();
+                return partyList.ToJObject();
+            });
```

Comment:
If ToJObject is so slow, should these be named like `getFullPartyListInfoSlow` or something obvious/obnoxious? Or are you planning on this not to exist forever and become something more targeted?

Reply:
> Ultimately I'm soliciting feedback and suggestions more on the ManagedType class and on the ways we can pull the structs in from the FFXIVClientStructs project.

I think `ManagedType` seems like a really good approach here that reduces the amount of code and maintenance.  Really nice work.

Re: pulling in structs.  Yeah, the more automated this can be the better.  My ideal version would be some sort of bot that could put up PRs to pull in new structs and you could just review/stamp it and merge.  There's just not a lot of folks maintaining OverlayPlugin, and so the easier we can make this, the better.

Re: which structs to pull in.  More code = more maintainer burden, imo.  I don't see value in pulling everything in just because it's there unless there's something there that somebody needs.  I think also there's something to be said about dalamud vs overlays.  If you're rewriting in game ui, you probably need all of this but you are probably also a dalamud plugin so you can do more intrusive things than just put an overlay over top of in game ui.  That's just my gut feeling here.